packages/shared/lib/contexts/governance/stores/proposal-filter.store.ts

```diff
@@ -0,0 +1,27 @@
+import { BooleanFilterOption } from '@core/utils/enums/filters'
+import { writable, Writable } from 'svelte/store'
+import { ProposalFilter, ProposalStatus, ProposalType } from '..'
+
+export const proposalFilter: Writable<ProposalFilter> = writable({
+    phase: {
+        active: false,
+        type: 'selection',
+        localeKey: 'filters.phase',
```

Comment:
Can we update this then to alleviate the burden on the translaters and to keep sure it's consistent across all languages? I guess the only thing that would need add is a label, right?

```suggestion
        localeKey: 'proposalStatus',
```